src/components/OfflineDetection.tsx

```diff
@@ -22,12 +20,10 @@ export const OfflineDetection = ({ children }: Props) => {
         `Connection status changed: connected=${state.isConnected}, reachable=${state.isInternetReachable}`,
       );
 
-      dispatch(
-        setNetworkInfo({
-          isConnected: state.isConnected,
-          isInternetReachable: state.isInternetReachable,
-        }),
-      );
+      setNetworkInfo({
+        isConnected: state.isConnected,
```

Comment:
I know it was already there, but could we rename `state` to `event` or something else here? just to make it 100% clear it has nothing to do with store state - thanks!